website/server/libs/cron.js

```diff
@@ -347,7 +393,7 @@ export function cron (options = {}) {
 
   // Login Incentives
   user.loginIncentives++;
-  user.addNotification('LOGIN_INCENTIVE', {});
+  awardLoginIncentives(user);
```

Comment:
I'd like this to be moved to earlier in this function to minimise the chance of it not being executed when bugs occur (e.g., it's not uncommon for `_.merge(progress, {down: 0, up: 0, collectedItems: 0}` to be skipped). People are really going to scream if they miss a count for their incentives.

It should also execute for people who are resting in the inn because that includes those who have chronic illnesses and they shouldn't be disadvantaged.

I'd also like `user.loginIncentives` to be added to analytics since this is the sort of thing people will log bug reports about if they think something's gone wrong. This might also be a good time to make `analytics.track('Cron' ...})` run when resting in the inn (see the TODO comment above).